src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [email, setEmail] = useState("");
+    const [birthday, setBirthday] = useState("");
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+
+        const data = {
+            Username: username,
+            Password: password,
+            Email: email,
+            Birthday: birthday
+        };
+
+        fetch("https://myflix-20778.herokuapp.com/login", {
+            method: "POST",
```

Comment:
Add the following line after `method: "POST"`

```
body: JSON.stringify(data)
```

Reply:
updated